service/application/src/main/java/com/solace/maas/ep/event/management/agent/service/ScanService.java

```diff
@@ -287,12 +242,21 @@ public Optional<ScanEntity> findById(String scanId) {
      * @param scanTypes          The scan types included in the scan request.
      * @param status             The status of scan.
      */
-    public void sendScanStatus(String groupId, String scanId, String traceId, String actorId, String messagingServiceId, String scanTypes,
+    public void sendScanStatus(String orgId,
+                               String groupId,
+                               String scanId,
+                               String traceId,
+                               String actorId,
+                               String messagingServiceId,
+                               String scanTypes,
                                ScanStatus status) {
+
+        Validate.notBlank(orgId, "Organization ID cannot be null or empty");
```

Comment:
I see that this error message have been used in many places. Can we add a CONSTANT for it plz?